packages/apputils/src/workspaces.ts

```diff
@@ -0,0 +1,16 @@
+import { ServerConnection, Workspace } from '@jupyterlab/services';
+
+import { IndexedDBDataConnector } from './statedb';
+
+export class WorkspaceManager extends IndexedDBDataConnector<Workspace.IWorkspace> {
+  constructor(options: IndexedDBDataConnector.IOptions) {
+    super({ ...options, storageName: 'JupyterLite Workspaces' });
+
+    this.serverSettings = ServerConnection.makeSettings();
```

Comment:
Maybe it should accept the `serverSettings?` in its options instead of creating new one? (for consistency, even though it's likely unused)